### PR TITLE
daemon/graphdriver/zfs fix lint errrors/warnings

### PR DESCRIPTION
**Pull request by brahmaroutu**
Addresses #14756
Signed-off-by: Srini Brahmaroutu srbrahma@us.ibm.com

Not sure if this package is already listed or not.

### CODE REVIEW

**Comment by calavera** [comment]
I think it needs to be added to the list.

File: daemon/graphdriver/zfs/zfs.go [hunk 1]
```diff
@@ -30,12 +30,17 @@ func init() {
 	graphdriver.Register("zfs", Init)
 }
 
+// Logger returns a zfs logger implmentation.
 type Logger struct{}
 
+// Log wrapps log message from ZFS driver with a prefix '[zfs]'.
```
Comment (vdemeester) [inline daemon/graphdriver/zfs/zfs.go]: s/wraps/wrapps/

File: daemon/graphdriver/zfs/zfs.go [hunk 2]
```diff
@@ -156,10 +162,15 @@ func (d *Driver) String() string {
...
 }
 
+// Cleanup is used to implement graphdriver.ProtoDriver. There is no cleanup required for this driver.
 func (d *Driver) Cleanup() error {
 	return nil
 }
 
+// Status return information about the ZFS filesystem. It returns a two dimensional array of information
+// such as pool name, dataset name, disk usage, parent quota and compression used.
+// Currently it return 'Zpool', 'Zpool Health', 'Parent Dataset', 'Space Used By Parent',
```
Comment (vdemeester) [inline daemon/graphdriver/zfs/zfs.go]: s/return/returns/

File: daemon/graphdriver/zfs/zfs.go [hunk 3]
```diff
@@ -189,6 +200,7 @@ func (d *Driver) Status() [][2]string {
 	}
 }
 
+// GetMetadata is used for implementing the graphdriver.ProtoDriver interface. VFS does not currently have any meta data.
```
Comment (vdemeester) [inline daemon/graphdriver/zfs/zfs.go]: Shouldn't it be ZFS instead of VFS here ? :sweat_smile:

File: daemon/graphdriver/zfs/zfs.go [hunk 4]
```diff
@@ -215,14 +227,17 @@ func (d *Driver) cloneFilesystem(name, parentName string) error {
 	return snapshot.Destroy(zfs.DestroyDeferDeletion)
 }
 
+// ZfsPath return the filesystem path for the id provided.
```
Comment (vdemeester) [inline daemon/graphdriver/zfs/zfs.go]: s/return/returns/

File: daemon/graphdriver/zfs/zfs.go [hunk 5]
```diff
@@ -215,14 +227,17 @@ func (d *Driver) cloneFilesystem(name, parentName string) error {
...
 func (d *Driver) ZfsPath(id string) string {
 	return d.options.fsName + "/" + id
 }
 
+// MountPath return the mounted filesystem path for the id provided.
```
Comment (vdemeester) [inline daemon/graphdriver/zfs/zfs.go]: s/return/returns/

**Comment by brahmaroutu** [comment]
@vdemeester  Update as per your comments.

**Comment by calavera** [comment]
LGTM

**Comment by icecrime** [comment]
LGTM

**Pull request merged**